src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.java

```diff
@@ -100,6 +105,14 @@ protected void onDisable() {
         mc.player.capabilities.allowFlying = false;
     }
 
+    private float getHighwaySpeed() {
```

Comment:
What lol why are there two different speeds

Reply:
One allows you to go past the set limit.